Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/openSAML/SAMLAssertionBuilderConstants.java

```diff
@@ -0,0 +1,10 @@
+package gov.hhs.fha.nhinc.callback.openSAML;
+
```

Comment:
Since this class deals with constant variable, can we move those two into NhincConstants.java?